packages/mobile/README.md

```diff
@@ -560,8 +560,8 @@ $ adb kill-server && adb start-server
 [rn profiler]: https://reactjs.org/blog/2018/09/10/introducing-the-react-profiler.html
 [rn running on device]: https://facebook.github.io/react-native/docs/running-on-device
 [setup]: ../../SETUP.md
-[react-native-testing-library]: https://github.com/callstack/react-native-testing-library
-[rntl-docs]: https://callstack.github.io/react-native-testing-library/
+[@testing-library/react-native]: https://github.com/callstack/react-native-testing-library
+[@tlrn-docs]: https://callstack.github.io/react-native-testing-library/
```

Comment:
nit: maybe we should keep the existing convention, without the `@`?

Reply:
I've removed the `@` from and changed the name back to `react-native-testing-library` as seen on their [npm page](https://www.npmjs.com/package/@testing-library/react-native). Also added links to `@testing-library/jest-native` in the testing description.